src/App.js

```diff
@@ -1,34 +1,20 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import { BrowserRouter, Route } from 'react-router-dom';
-import { connect } from 'react-redux';
 
 import { Header } from './components/Header/Header';
 import { Home } from './components/Home/Home';
 import { Game } from './components/Game/Game';
 import { Result } from './components/Game/Result/Result';
 
-const AppView = ({ words, answer }) => (
+const App = () => (
     <BrowserRouter basename="/scrabble-workout">
         <div>
             <Header />
             <Route exact path="/" component={Home} />
             <Route exact path="/game" component={Game} />
-            <Route
-                path="/game/result"
-                /* eslint-disable-next-line react/jsx-props-no-spreading */
-                render={(props) => <Result {...props} words={words} answer={answer} />}
-            />
+            <Route path="/result" component={Result} />
```

Comment:
this is the smallest thing but `game` route needs `exact` prop as much as `result` route but only `game` route has it